maisim/maisim.Game/Beatmaps/Beatmap.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace maisim.Game.Beatmaps
+{
+    /// <summary>
+    /// Class that represents a beatmap that includes all the information needed on beatmap.
+    /// </summary>
+    public class Beatmap
```

Comment:
This should use properties instead of encapsulated privates properties and a constructor

Reply:
Yes it's shold be like that.